.github/workflows/bot-pr-updated.yml

```
@@ -60,3 +60,14 @@ jobs:
         with:
             name: abi
             path: abi.diff
+  wiki-check:
```

nit: A bit more descriptive
```suggestion
  changelog-wiki-check:
```

---

I use check-wiki-changelog because others also use verb first